xpk.py

```diff
@@ -1180,12 +1230,12 @@ def run_gke_node_pool_create_command(args, system_characteristics) -> int:
   Returns:
     0 if successful and 1 otherwise.
   """
-
+  device_type = args.tpu_type if args.tpu_type else args.gpu_type
   xpk_print(
-      f'Creating {args.num_slices} node pool or pools of {args.tpu_type}\n'
+      f'Creating {args.num_slices} node pool or pools of {device_type}\n'
       f'Underlyingly, we assume that means: {system_characteristics}'
   )
-
+  print("system_char is ", system_characteristics)
```

Comment:
Is this a debug/temporary print statement?

Reply:
yes, removed it in newer commit